src/server/redis_connection.cc

```diff
@@ -130,6 +130,39 @@ void Connection::Reply(const std::string &msg) {
   redis::Reply(bufferevent_get_output(bev_), msg);
 }
 
+std::string Connection::Bool(bool b) const {
+  if (protocol_version_ == RESP::v3) {
+    return b ? "#t" CRLF : "#f" CRLF;
+  }
+  return Integer(b ? 1 : 0);
```

Comment:
In RESPv2, the `false` will be mapped to `0` or `nil`?

Reply:
Only one place will map false to nil, see: https://github.com/apache/kvrocks/blob/e239e83d55dd2ece43f8b1cd8cbd2a5e44653b42/src/storage/scripting.cc#L1089

Reply:
Ahh I see. I wonder what the behavior is in Redis?

Reply:
Yes, I have confirmed this while changing this line.

Reply:
Got it.